src/main/java/org/folio/linked/data/configuration/CacheConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.linked.data.configuration;
+
+import java.util.Collections;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.concurrent.ConcurrentMapCache;
+import org.springframework.cache.support.SimpleCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CacheConfiguration {
```

Comment:
Let's try a solution provided by Spring Cloud for Feign Client?
https://docs.spring.io/spring-cloud-openfeign/docs/current/reference/html/#feign-caching

Reply:
done